containers/phdi-ingestion/app/routers/cloud_storage.py

```diff
@@ -17,6 +17,7 @@ class WriteBlobToStorageInput(BaseModel):
     cloud_provider: Optional[Literal["azure", "gcp"]]
     bucket_name: Optional[str]
     file_name: str
+    storage_account_url: Optional[str] = None
```

Comment:
Really a nit, but what do think about this? I'd argue for setting the default value to an empty string instead of `None` so that it will be of type string which is consistent with our typing hinting.

```suggestion
storage_account_url: Optional[str] = ""
```

Here is a demonstration of the type difference between `None` and `""`.

```
Python 3.10.6 (main, Aug  8 2022, 19:28:58) [Clang 13.1.6 (clang-1316.0.21.2.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> type(None)
<class 'NoneType'>
>>> type("")
<class 'str'>
```